eppo_client/client.py

```diff
@@ -118,7 +169,13 @@ def _get_subject_variation_override(
             experiment_config is not None
             and subject_hash in experiment_config.overrides
         ):
-            return experiment_config.overrides[subject_hash]
+            override_variation = VariationDto(
+                name="override",
+                value=experiment_config.overrides[subject_hash],
+                typedValue=experiment_config.typedOverrides[subject_hash],
```

Comment:
Assumption that the `overrides` and `typedOverrides` dicts will always contain identical sets of keys